src/commands.rs

```diff
@@ -58,32 +88,62 @@ pub struct Install {
     after_help = "For more information, read the README.md",
     override_usage = "soldeer update"
 )]
-pub struct Update {}
+pub struct Update {
+    #[arg(long, value_parser = clap::value_parser!(bool))]
+    /// This regenerates the remappings from scratch, will delete old remappings and regenerate
+    /// them
+    #[clap(
+        help = "Use this option set to true if you want to regenerate the remappings from scratch. This will delete the old remappings."
+    )]
```

Comment:
Usually, I like to skip the `clap(help)` altogether and put the help text inside the doc comment. these get parsed by clap so it avoids duplication and looks sleeker in the code.